location-service/src/main/resources/db/migrations/V1__Create_location_table.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE IF NOT EXISTS locations (
+    id BIGSERIAL PRIMARY KEY,
+    latitude DOUBLE PRECISION NOT NULL,
+    longitude DOUBLE PRECISION NOT NULL,
+    user_id BIGINT UNIQUE NOT NULL,
+    created_at TIMESTAMP NOT NULL
+);
```

Comment:
add index on user_id field, to improve query performance for frequent lookups by user_id